src/NetAnalyzers/Core/Microsoft.CodeQuality.Analyzers/ApiDesignGuidelines/UseGenericEventHandlerInstances.Fixer.cs

```diff
@@ -1,15 +1,18 @@
 // Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
+using Microsoft.CodeAnalysis;
 using Microsoft.CodeAnalysis.CodeFixes;
 using System.Collections.Immutable;
+using System.Composition;
 using System.Threading.Tasks;
 
 namespace Microsoft.CodeQuality.Analyzers.ApiDesignGuidelines
 {
     /// <summary>
     /// CA1003: Use generic event handler instances
     /// </summary>
-    public abstract class UseGenericEventHandlerInstancesFixer : CodeFixProvider
+    [ExportCodeFixProvider(LanguageNames.CSharp, LanguageNames.VisualBasic), Shared]
+    public sealed class UseGenericEventHandlerInstancesFixer : CodeFixProvider
```

Comment:
We should probably just delete this file - it wouldn't be hard to add it back if/when we add a fixer.